src/styles/blocks/calendar.scss

```
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1px;
```

Используется в нескольких местах, лучше использовать значение, как переменную
```suggestion
  gap: 1px;
```